process_pr.py

```diff
@@ -1003,6 +1014,21 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
                     pull_request_updated = True
             continue
 
+        if (
+            (commenter == cmsbuild_user)
+            and re.match(CMSBOT_TECHNICAL_MSG, first_line)
+            and not technical_comment
+        ):
+            technical_comment = comment
+
+        if (commenter == cmsbuild_user) and "This PR contains too many commits" in first_line:
+            warned_too_many_commits = True
+
+        if commenter in l2s and re.match(
```

Comment:
@iarspider , to match the exact github user I think it is better to use `commenter in CMSSW_ISSUES_TRACKERS` instead of ` commenter in l2s`